Makefile

```diff
@@ -250,12 +250,17 @@ docs-container:
 	echo ".*" >>.dockerignore # .git pruned out
 	docker image build -t cilium/docs-builder -f Documentation/Dockerfile .
 
+
 render-docs: docs-container
 	-docker container rm -f docs-cilium >/dev/null
 	docker container run -ti -u $$(id -u):$$(id -g $(USER)) -v $$(pwd):/srv/ cilium/docs-builder /bin/bash -c 'make html' && \
 	docker container run -dit --name docs-cilium -p 8080:80 -v $$(pwd)/Documentation/_build/html/:/usr/local/apache2/htdocs/ httpd:2.4
 	@echo "$$(tput setaf 2)Running at http://localhost:8080$$(tput sgr0)"
 
+test-docs: docs-container
+	-docker container rm -f docs-cilium >/dev/null
+	docker container run --rm -v $$(pwd):/srv/ cilium/docs-builder /bin/bash -c 'make html'
```

Comment:
Does this command ever return a non-zero return code? I.e., will the build fail if there is an error in the docs? Or is the onus upon the user who does doc changes to check the build output and fix them?

Reply:
the build fail yes